### PR TITLE
Merge Testing into Main

### PR DESCRIPTION
**Pull request by bhavberi**

### CODE REVIEW

File: experiment/aim.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
-A variable of a certain data type such as int or char can store only one element of that data type. An array, on the other hand, can store multiple elements of a fixed data type. An array, thus, allows a programmer to address a collection of similar elements by a single name, while also allowing him to address each of the elements by their index in the array.
+When working with data in programming, you often encounter variables of specific data types like int or char, which can store only a single element of that data type. However, the real world is rarely so simplistic. What if you need to manage a collection of related elements, each with its own data type? Arrays offer a solution, allowing you to store multiple elements of a fixed data type. With arrays, you can address this collection collectively using a single name while accessing each individual element through its unique index in the array.
```
Comment (raj-vlabs) [inline experiment/aim.md]: _'What if you need to manage a collection of related elements, each with its own data type? Arrays offer a solution'_
Arrays do not support different data types in one collection

File: experiment/aim.md [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+But what if you want to create a collection of data elements with diverse data types and reference this entire ensemble with a single identifier? The answer is a resounding 'yes,' and the tool for this task is structures. Structures empower you to assemble a heterogeneous collection of elements, each with its own data type, and then reference this entire assembly using a single name. Structures can even include arrays, making them incredibly versatile.
 
+In essence, a structure serves as a custom data type that can encompass multiple components of various data types, allowing you to encapsulate complex data structures in a cohesive manner. This flexibility also extends to creating arrays of structure variables, further enhancing your ability to manage and organize your data effectively.
+
+With this lab, you will gets hands-on and try:
```
Comment (raj-vlabs) [inline experiment/aim.md]: It's an experiment not a lab

File: experiment/posttest.json [hunk 3]
```diff
@@ -1,30 +1,136 @@
...
+                "a": "10 60",
+                "b": "20 60",
+                "c": "40 50",
+                "d": "80 30"
+            },
+            "correctAnswer": "a",
+            "difficulty": "intermediate"
+        },
+        {
+            "question": "2. Can arrays be a part of the structure?",
```
Comment (raj-vlabs) [inline experiment/posttest.json]: Grammatically correct the sentence.
An Array can be a part of a Structure

File: experiment/simulation/index.html [hunk 4]
```diff
@@ -1,48 +1,92 @@
...
+							<button class="button buttonPosition" id='submitBtnId'>Submit Code</button>
+							<!-- <button id='submitBtnId1'>Submit Code</button> -->
+						</div>
+						<div class="resultDisplay">
+							<textarea class="codeEditArea" id='solutionHintArea' dis></textarea>
+						</div><!-- end of resultDisplay -->
+					</div>
+				</div>
+			</div>
+			<!-- <div class="mdl-cell mdl-cell--6-col mdl-cell--8-col-tablet mdl-cell--4-col-phone">
```
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Please remove all commented  out code

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
1. Remove all commented out code.
2. Check the questions for grammar and consistency.

File: experiment/posttest.json (hunk 3 above)
Comment (bhavberi) [reply experiment/posttest.json]: I didn't create these questions; only a few at the end were made by me. I didn't check at that time, but I have changed it now.

File: experiment/simulation/js/main.js [hunk 5]
```diff
@@ -1 +1,345 @@
...
+// Email: <bhavberi@gmail.com>    |
+//---------------------------------+
+
+window.model = {
+
+}
+
+window.view = {
+    stateOfSolution: 1, // shows state of solution.
+    stateOfExeplation: 0, // shows state of exeplation.
```
Comment (raj-vlabs) [inline experiment/simulation/js/main.js:15]: What is exeplation? Is it explanation misspelt?

**Review by raj-vlabs — APPROVED** [review]

**Pull request merged**